code/drasil-lang/Language/Drasil/Chunk/DataDefinition.hs

```diff
@@ -38,13 +39,14 @@ instance NamedIdea          DataDefinition where term = qd . term
 instance Idea               DataDefinition where getA c = getA $ c ^. qd
 instance HasSpace           DataDefinition where typ = qd . typ
 instance HasSymbol          DataDefinition where symbol e st = symbol (e^.qd) st
-instance Quantity           DataDefinition where getUnit (DD a _ _ _ _ _) = getUnit a
+instance Quantity           DataDefinition where 
 instance ExprRelat          DataDefinition where relat = qd . relat
 instance HasReference       DataDefinition where getReferences = ref
 instance Eq                 DataDefinition where a == b = (a ^. uid) == (b ^. uid)
 instance HasDerivation      DataDefinition where derivations = deri
 instance HasAdditionalNotes DataDefinition where getNotes = notes
 instance HasShortName       DataDefinition where shortname = view lbl
+instance MayHaveUnit        DataDefinition where getUnit (DD a _ _ _ _ _) = getUnit a 
```

Comment:
In this case, the old code was bad. You should use the lens `qd` to get at this.